docs/_portfolios/juzibot.md

```diff
@@ -19,13 +19,15 @@ ventures:
 
 国内领先的智能对话服务商，专注于将智能对话落地到微信等社交生态，通过设计专业的对话系统对接流程，将业务场景梳理、数据收集、对话脚本撰写、系统搭建、评估反馈等进行标准化，帮助企业搭建符合真实业务场景的对话机器人。
 
-句子互动的客服和数据分析系统，为企业赋能，大幅度增强其获客、用户粘性和销售转化的能力。客户来自教育、保险、旅游等多个领域，包括好未来、中国人寿、黄包车旅行、美团点评等，产品服务客户规模覆盖年销售额 100 万至 10 亿。其中 80% 的客户来自口碑传播，每天提升 100 万人次的销售体验。
+句子互动围绕微信生态为客户提供的[智能营销和销售的SaaS工具](https://juzibot.com)，帮助企业搭建安全稳定的私域流量池，为企业赋能，大幅度增强其获客、用户粘性和销售转化的能力，助力企业客户引流并实现转化，客户覆盖教育、保险、大健康等多个领域，包括美团点评、神州数码、作业帮等, 其中 80% 的客户来自口碑传播，每天提升 100 万人次的销售体验。
 
 先后获得 PreAngel、Plug and Play、Y Combination、TSVC（清谷资本）和阿尔法公社等多家中美机构天使投资。
 
 - 2017 年入选百度AI加速器，并于次年和百度联手打造[《从0到1搭建聊天机器人》智能对话平台(UNIT)系列课程](https://ai.baidu.com/support/video)
 - 2018 年荣获 CCFA 零售技术新锐企业奖
 - 2019 年成为腾讯云智能对话平台合作伙伴
 - 2018、2019 年分别在中国中文信息学会举办的SMP中文人机对话技术评测中，荣获了任务型人机对话回复自然度第一名，和闲聊型人机对话测评第一名的好成绩
+- 2020 年入选 Facebook 加速器
+
+公司文化中拥有浓厚的开源精神，相信通过互联网开放式协作可以构建更美好的世界。是 Github 上 6,000+ Stars 知名开源 Chatbot 项目 Wechaty 的黄金赞助伙伴，基于开源项目 Wechaty （Github 7000+ star）运营了一个覆盖全球最具影响力的微信机器人开发者社区，覆盖上万名开发者，其中很多来自于 Microsoft、Google、腾讯等百余家巨头高科技企业，地区分布在美国、欧洲、中国、加拿大、澳大利亚、英国、中国香港、中国台湾国家和地区。
```

Comment:
> 是 Github 上 6,000+ Stars 知名开源 Chatbot 项目 Wechaty 的黄金赞助伙伴，基于开源项目 Wechaty （Github 7000+ star）

I believe those descriptions are duplicated. (6,000 and 7000)

Reply:
fix